programs/wen-new-standard/src/instructions/manager/init.rs

```diff
@@ -0,0 +1,23 @@
+use anchor_lang::prelude::*;
+
+use crate::{Manager, MANAGER_SEED};
+
+#[derive(Accounts)]
+#[instruction()]
+pub struct InitManagerAccount<'info> {
+    #[account(mut)]
+    pub payer: Signer<'info>,
+    #[account(
+        init,
+        payer = payer,
+        seeds = [MANAGER_SEED],
+        space = Manager::LEN,
```

Comment:
Transition to InitSpace macro. Will need to happen throughout but can start with this function. https://www.anchor-lang.com/docs/space#the-init-space-macro